src/main/filesystem.ts

```diff
@@ -76,9 +77,12 @@ function createDirIfNotExists(path: fs.PathLike) {
   }
 }
 
-
-export function breakingAppVersion(app: Electron.App): string {
+export function breakingAppVersion(): string {
   const version = app.getVersion();
+  return breakingVersion(version);
+}
+
+export function breakingVersion(version: string): string {
```

Comment:
Do we need two functions for this, I think this can be consolidated into one

Reply:
I think separate functions are useful in this case because I'm using the `breakingVersion()` function to check version compatibility of releases on github when fetching available updates but I also want to check the version of the running app in other places for which I can use `breakingAppVersion()`.